### PR TITLE
add loading content and getX for state management

### PR DESCRIPTION
**Pull request by lordvins226**
#2

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -211,6 +212,7 @@ Please take a quick look at the [contribution guidelines](CONTRIBUTING.md) first
 * [get_it_mixin](https://pub.dev/packages/get_it_mixin) - mixin on Stateless- and StatefulWidgts that makes binding the Widget to Objects stored in GetIt easier. Makes get_it to a full state management solutio
 * [mobx](https://pub.dev/packages/mobx) - MobX is a state-management library that makes it simple to connect the reactive data of your application with the UI.
 * [provider](https://pub.dev/packages/provider) - A wrapper around InheritedWidget to make them easier to use and more reusable.
+* [getX](https://pub.dev/packages/get) - GetX is an extra-light and powerful solution for Flutter. It combines high performance state management, intelligent dependency injection, and route management in a quick and practical way.
```
Comment (polilluminato) [inline README.md]: As described in the contribution.md could you please insert the package in alphabetic order?

File: README.md [hunk 2]
```diff
@@ -27,6 +27,7 @@ Please take a quick look at the [contribution guidelines](CONTRIBUTING.md) first
     - [Timeline](#timeline)
     - [Utils](#utils)
     - [Video Player](#video-player)
+    - [Loading](#loading)
```
Comment (polilluminato) [inline README.md]: As described in the contribution.md at least 3 items are needed to create a new category. If you find at least two others you can easily create a new category

**Review by polilluminato — CHANGES_REQUESTED** [review]
Check my comment on the commit.

File: README.md (hunk 1 above)
Comment (lordvins226) [reply README.md]: Ok I will make it

**Pull request closed without merging**